katuma.yml

```diff
@@ -14,3 +14,14 @@
         log_path: "{{ shared_path }}/log"
         pids_path: "{{ shared_path }}/pids"
         ofn_route_override: "/admin/reports/order_cycle_management"
+
+    - role: jdauphant.nginx
+      nginx_sites:
+        katuma_redirect:
+          - listen 443 ssl http2
+          - listen [::]:443 ssl http2
+          - server_name staging.katuma.org
+          - return 301 https://appstaging.katuma.org$request_uri
+
+          - ssl_certificate      /etc/letsencrypt/live/staging.katuma.org/fullchain.pem
+          - ssl_certificate_key  /etc/letsencrypt/live/staging.katuma.org/privkey.pem
```

Comment:
Now that I found https://github.com/jdauphant/ansible-role-nginx#9-site-configuration-using-a-custom-template, it's better to replace it.